package.json

```diff
@@ -21,27 +21,24 @@
   },
   "homepage": "https://github.com/firecow/git-local-devops#readme",
   "dependencies": {
+    "ajv": "^8.11.0",
     "await-to-js": "^3.0.0",
     "chalk": "^4.1.2",
     "dotenv": "^16.0.0",
     "fs-extra": "^10.0.1",
     "js-yaml": "^4.1.0",
-    "promisify-child-process": "^4.1.1",
+    "ts-node": "^10.7.0",
```

Comment:
@LauJosefsen Isn't this an dev-dep ?

Reply:
Has been moved.

Also I think we need to consider having a build script for publishing, but that can be solved in another PR, as this is big enough for now? :P